apps/webapp/app/presenters/OrganizationsPresenter.server.ts

```diff
@@ -2,13 +2,16 @@ import { PrismaClient } from "@trigger.dev/database";
 import { redirect } from "remix-typedjson";
 import { prisma } from "~/db.server";
 import {
+  clearCurrentProjectId,
   commitCurrentProjectSession,
   getCurrentProjectId,
   setCurrentProjectId,
 } from "~/services/currentProject.server";
 import { logger } from "~/services/logger.server";
 import { newProjectPath } from "~/utils/pathBuilder";
 import { ProjectPresenter } from "./ProjectPresenter.server";
+import { redirectWithErrorMessage } from "~/models/message.server";
+import { match } from "assert";
 
 export class OrganizationsPresenter {
```

Comment:
This file deals with selecting a project, as well as organizations. This is because we always have a selected project due to the nature of our sidebar/design.

Where possible we now redirect users to the correct page rather than just showing them an error.